airflow/operators/bash_operator.py

```diff
@@ -18,26 +18,18 @@
 # under the License.
 
 
-from builtins import bytes
 import os
 import signal
 from subprocess import Popen, STDOUT, PIPE
 from tempfile import gettempdir, NamedTemporaryFile
 
-from airflow import configuration as conf
+from builtins import bytes
+
 from airflow.exceptions import AirflowException
 from airflow.models import BaseOperator
 from airflow.utils.decorators import apply_defaults
 from airflow.utils.file import TemporaryDirectory
-
-
-# These variables are required in cases when BashOperator tasks use airflow specific code,
-# e.g. they import packages in the airflow context and the possibility of impersonation
-# gives not guarantee that these variables are available in the impersonated environment.
-# Hence, we need to propagate them in the Bash script used as a wrapper of commands in
-# this BashOperator.
-PYTHONPATH_VAR = 'PYTHONPATH'
-AIRFLOW_HOME_VAR = 'AIRFLOW_HOME'
```

Comment:
@yrqls21 Could you give me some context on why changing this file is needed as part of this PR? (same holds for the tests)

Reply:
These are mainly to cover the cases mentioned in the description. Changes in bash_operator.py is to cover the case that `hive` is called in a bash script and changes in python_operator.py is to cover the  cases that Hive hooks are called in a python callable. We see both cases( though rare) in Airbnb and thus want to add them to get bigger coverage.